js/src/util/sanitizer.js

```diff
@@ -19,23 +19,19 @@ const uriAttributes = new Set([
 /**
  * A pattern that recognizes a commonly useful subset of URLs that are safe.
  *
- * Shout-out to Angular https://github.com/angular/angular/blob/12.2.x/packages/core/src/sanitization/url_sanitizer.ts
+ * Shout-out to Angular https://github.com/angular/angular/blob/5a37928babc1eecaf66bf67f9678f64ed388c98a/packages/core/src/sanitization/url_sanitizer.ts#L38
  */
-const SAFE_URL_PATTERN = /^(?:(?:https?|mailto|ftp|tel|file|sms):|[^#&/:?]*(?:[#/?]|$))/i
+// eslint-disable-next-line unicorn/better-regex
```

Comment:
There was a lint rule that offered to optimize the regex.  I disabled it temporarily to get some feedback because it'll be slightly different than the one from angular.  Should I go with the optimized version or did you all want to stay close to the angular regex?

Reply:
New regex `/^(?!javascript:)(?:[\d+.a-z-]+:|[^#&/:?]*(?:[#/?]|$))/i`

Reply:
It's OK, better stay in sync since the changes are pretty minor anyway